integreat_cms/cms/templates/pages/page_form_sidebar/settings_box.html

```diff
@@ -20,7 +20,7 @@
         {% render_field page_form|get_private_member:"ref_node_id" %}
         {% render_field page_form|get_private_member:"position" %}
         {% render_field page_form.parent id="parent" %}
-        <label class="btn-gray">{% translate "Order" %}</label>
```

Comment:
Could you also delete the definition in the stylesheet? So we don't get tempted to use this again in the future :wink: